README.md

```diff
@@ -107,17 +108,16 @@ headers:
     <td>Filters for declarations.<br><b>Default: all are included</b></td>
     <td><pre lang="yaml"><code>
 functions:
-  include: # Exclude is also available.
-    names: # Matches with exact name.
-      - someFuncName
-      - anotherName
-    matches: # Matches using regexp.
-      - prefix.*
-      - [a-z][a-zA-Z0-9]*
-  prefix: 'cx_' # Prefix added to all functions.
-  prefix-replacement: # Replaces a functions's prefix.
-    'clang_': ''
-    '_': 'C'</code></pre></td>
+  include: # 'exclude' is also available.
+    - [a-z][a-zA-Z0-9]* # Matches using regexp.
+    - prefix.* # '.' matches any character.
```

Comment:
Wouldn't this also match `blablablaprefix.cheese`? because there is no anchors? From what I see you're using `firstMatch`..

Reply:
Are anchors really required?
I am doing a fullMatch [(package:quiver](https://pub.dev/packages/quiver))
- For include/exclude -> https://github.com/mannprerak2/ffigen/blob/declaration_include_exclude_rename/lib/src/config_provider/config_types.dart#L183
 - For renaming, I first see if its a full match, and then use firstMatch (which should return the largest match I suppose)
https://github.com/mannprerak2/ffigen/blob/declaration_include_exclude_rename/lib/src/config_provider/config_types.dart#L126

Also we match with declaration names, which in C can only contain `_, a-z, A-Z, 0-9`

Reply:
Here is the function used for full matching (in package quiver)
```dart
bool matchesFull(Pattern pattern, String str) {
  var match = pattern.matchAsPrefix(str);
  return match != null && match.end == str.length;
}
```

Reply:
> Are anchors really required?

No probably not, my bad :D